website/docs/reference/sdks/flutter-proxy.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Should we change the file name to just `flutter.md`? That would align better with the preexisting SDKs. Any reason not to do that?

Reply:
I followed the android-proxy and ios-proxy conventions.